server/routes.go

```diff
@@ -497,6 +497,41 @@ func (b *BaseRouter) addActivityRoutes() {
 		}
 		c.AbortWithStatusJSON(http.StatusBadRequest, ErrorResponse{Error: err.Error()})
 	})
+
+	activity.PUT(":id", func(context *gin.Context) {
+		userId := context.MustGet("userId").(uint)
+		id, err := strconv.ParseUint(context.Param("id"), 10, 32)
+		if err != nil {
+			context.Status(400)
```

Comment:
In places where we just return a 400 status (no error msg returned to user), it may be beneficial to add an error log.

If we ever got an unexpected 400 in the future, it would be a big help to see the error in the server log.